scss/os/_utils.scss

```diff
@@ -590,6 +594,11 @@ $somesizes: map-merge((
   top: -1.3rem;
 }
 
+.u-progress__large {
+  font-size: 1rem;
```

Comment:
I think it would be best if these values were multipliers on the original variables, so `$progress-height * <multiplier>` and `$progress-font-size * <multiplier>`. That would ensure the progress bar is still larger if one were to change those variables.